### PR TITLE
Add integration script and workflow for tdx server

### PR DESCRIPTION
**Pull request by liuw1**
.yml file is implemented by Arron.

Signed-off-by: Arron Wang <arron.wang@intel.com>
Signed-off-by: Wei Liu <wei3.liu@intel.com>

### CODE REVIEW

**Review by jyao1 — APPROVED** [review]

File: sh_script/integration_tdx.sh [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+guest_image="/home/env/guest_img/td-guest-centos8.4.qcow2"
+kernel="/home/env/kernel_img/bzImage"
+cloud_hypervisor_tdx_path="/home/env/cloud-hypervisor/target/release/cloud-hypervisor"
+
+firmware=""
+type="pe"
+
+# Test Configuration Info
+cpus=1
```
Comment (jiangliu) [inline sh_script/integration_tdx.sh:20]: Should we change this to 2 for smp related testing?

Comment (liuw1) [reply sh_script/integration_tdx.sh:20]: Hi Jiang, this is initial version of TDX CI, I will update it this week to contain multiple processors testing

Comment (liuw1) [reply sh_script/integration_tdx.sh:21]: I have update 3 configurations: 
cpus 1 + memory 2G
cpus 2 + memory 4G
cpus 3 + memory 8G

**Comment by jyao1** [comment]
Integrate feedback from https://github.com/confidential-containers/td-shim/pull/221

**Pull request merged**